### DISCUSSION TITLE
S-132 hiccup

### DISCUSSION THREAD

**Discussion by MarkusPlehn** [thehighnibble/thehighnibble.github.io#171] [S-132 add-on for IMSAI & Cromemco replicas]
i tested the S-132 with "WARGAMES" (CP/M disk) and there is a "problem" .
In game you can use a dialer and "call telefon numbers". when you try to dial 1000 numbers or more, they will displayed page by page and sometimes it comes a "wait" and "host found" from S-132 and then go on with number pages.
the tty console at the IMSAI8080 Desktop has no problems with displaying page by page

is it a connection or data problem? it seems losing the connection for a short moment, or buffer overflow ??

**Comment by dmcnaugh** [comment]
Looks like the S-132 is rebooting. 
If I can reproduce the problem then I can fix it.
Can you send a link to the "WARGAMES" disk image you are using, and tell me the steps to reproduce the error.

Also, it might be related to the _Scroll History_ feature of the VT-100, so can you disable this and see if the problem continues.
On the `Quick Menu -> Display -> Lines of history -> Disable`

**Reply by MarkusPlehn** [reply]
scroll history I trying before, no different.
I will send you the " .dsk" via mail .

**Comment by dmcnaugh** [comment]
The VT-100 emulation is working as it should.

The _WARGAMES -> DIALER_ application is sending `ESC c` at the start of every page.

You can see this for yourself if you set: `Quick Menu > Display > Show control characters`

If you refer to https://vt100.net/docs/vt510-rm/RIS the RIS control sequence is a full terminal reset. 

The page says:

```
It is recommended that you not use RIS to reset the terminal.
You should use a soft terminal reset (DECSTR) instead. 
RIS usually causes a communication line disconnect and may change the current baud rate settings. 
When performing a RIS, the terminal sends XOFF to the host to stop communication. 
When the RIS is complete, the terminal sends XON to resume communication.
```

You should ask the author of the WARGAMES demo/application to modify their code not to send **RIS**, `ESC c`
